.travis.yml

```diff
@@ -8,7 +8,7 @@ jdk:
 # Caching per https://docs.travis-ci.com/user/languages/java/
 before_cache:
   - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
-  - rm -f $HOME/.gradle/caches/2.9/plugin-resolution/cache.properties.lock
+  - rm -f $HOME/.gradle/caches/3.5/plugin-resolution/plugin-resolution.lock
```

Comment:
Just that TravisCI told me to 😃  (Though I see they've updated their documentation since I originally put that in.)

https://docs.travis-ci.com/user/languages/java/#Caching